.changeset/ninety-toys-smile.md

```diff
@@ -0,0 +1,5 @@
+---
+'xstate': patch
+---
+
+Added TTypestate to withConfig, state, and start methods
```

Comment:
could you mention what does this fix? this message will appear in our changelog and right now it might be a little bit vague for a reader